groupie/src/test/java/com/genius/groupie/UpdatingGroupTest.java

```diff
@@ -0,0 +1,53 @@
+package com.genius.groupie;
+
+import android.databinding.ViewDataBinding;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.runners.MockitoJUnitRunner;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.verifyNoMoreInteractions;
+
+@RunWith(MockitoJUnitRunner.class)
+public class UpdatingGroupTest {
+
+    @Mock GroupAdapter groupAdapter;
+
+    class AlwaysUpdatingItem extends Item {
+        @Override public int getLayout() { return 0; }
+        @Override public void bind(ViewDataBinding viewBinding, int position) {}
+
+        public AlwaysUpdatingItem(int id) {
+            super(id);
+        }
+
+        @Override
+        public boolean equals(Object obj) {
+            return false;
+        }
+    }
+
+    @Test
+    public void john() {
```

Comment:
I'm guessing you just called it quits for the night, but I'd call this test something meaningful.  Otherwise, you rock